shared/js/bluetooth_helper.js

```diff
@@ -252,6 +268,18 @@ var BluetoothHelper = function() {
       });
     },
 
+    set ona2dpstatuschanged(callback) {
+      _ready(function() {
+        _adapter.ona2dpstatuschanged = callback;
+      });
+    },
+
+    set onrequestmediaplaystatus(callback) {
+      _ready(function() {
+        _adapter.onrequestmediaplaystatus = callback;
+      });
+    },
+
```

Comment:
Looks like these two methods are added because the remote controls module needs them, so I guess the bluetooth helper is still missing some methods if it wants to play a good role as a helper of bluetooth.

I checked the Bluetooth document on MDN: https://developer.mozilla.org/en-US/docs/Web/API/BluetoothAdapter , it seems like we should probably add "BluetoothAdapter.ondevicefound" into this bluetooth helper, so that it can cover all the use cases if any other apps need it. Are there more in the V2 bluetooth api? the document is probably for V1 only.